src/main/java/com/zaxxer/nuprocess/internal/LibJava10.java

```diff
@@ -71,5 +72,5 @@ public static native int Java_java_lang_ProcessImpl_forkAndExec(
            Object dir,
            Object fds,
            byte redirectErrorStream
-   );
+   ) throws IOException;
```

Comment:
This matches [Java's own declaration](https://github.com/JetBrains/jdk8u_jdk/blob/master/src/solaris/classes/java/lang/UNIXProcess.java#L208-L215), which helps make it more obvious when `LinuxProcess` calls this method that it can throw.